shell_integration/windows/OCContextMenu/OCContextMenu.cpp

```diff
@@ -72,7 +72,7 @@ IFACEMETHODIMP_(ULONG) OCContextMenu::Release()
 
 // Initialize the context menu handler.
 IFACEMETHODIMP OCContextMenu::Initialize(
-    LPCITEMIDLIST pidlFolder, LPDATAOBJECT pDataObj, HKEY hKeyProgID)
+    LPCITEMIDLIST, LPDATAOBJECT pDataObj, HKEY)
```

Comment:
We probably won't find a consensus here ...
but I would like to point out that I think it's a bad idea to remove parameter names here ... I would prefer `Q_UNUSED` or adding `/* */ ` around the parameter names.

Reply:
Anyone who touches the code will need to open the doc anyhow